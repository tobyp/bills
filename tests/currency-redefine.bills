#!./bills

A	B.	4.00

%set XTS 2.0  # XTS is the test currency code

A	B.	2.00*XTS

%set XTS 4.0

A	B.	1.0*XTS
