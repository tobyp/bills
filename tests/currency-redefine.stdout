todo: B pays 12.0 to A
